pkg/build/registry/buildconfig/webhook.go

```diff
@@ -72,9 +77,18 @@ func (w *WebHook) ServeHTTP(writer http.ResponseWriter, req *http.Request, ctx k
 		Revision:    revision,
 		Env:         envvars,
 	}
-	if _, err := w.instantiator.Instantiate(config.Namespace, request); err != nil {
+	newBuild, err := w.instantiator.Instantiate(config.Namespace, request)
+	if err != nil {
 		return errors.NewInternalError(fmt.Errorf("could not generate a build: %v", err))
 	}
+
+	// Send back the build name so that the client can alert the user.
+	if newBuildEncoded, err := runtime.Encode(kapi.Codecs.LegacyCodec(w.groupVersion), newBuild); err != nil {
+		glog.Warningf("build \"%s\" was started, but could not encode response object: %v", newBuild.Name, err)
```

Comment:
This is pretty bad.  `utilruntime.handleError` so sentry smacks us and we go find you :)